### PR TITLE
chore: deprecate-synths

### LINKED ISSUE #1957: Exchange: Remove deprecated synths
### Subject of the issue
Synthetix deprecated several synths with SIP-293, therefore we need to clean up our Exchange.

### Deliverables

**On Optimism L2:**
- Remove deprecated synths (`sAAVE, sAVAX, sLINK, sMATIC, sSOL, sUNI`) from the Exchange dropdown options
- Move remaining synths into the "Tokens" list
- Remove "Synths" header in the dropdown

### Bounty
50 $OP

### Deadline 
Feb 6, 2023

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
## Description
Remove deprecated synths

## Related issue
fixes #1957

## Screenshots (if appropriate):

Screenshot 2023-02-04 at 1 18 09 AM

### CODE REVIEW

File: sdk/data/synths.ts [hunk 1]
```diff
@@ -160,39 +137,6 @@ export const synths: Record<SynthSymbol, BasicSynth> = {
 		},
 		category: 'forex',
 	},
-	sLINK: {
-		name: 'Chainlink',
-		asset: 'LINK',
-		addresses: {
-			1: '0xbBC455cb4F1B9e4bFC4B73970d360c8f032EfEE6',
```
Comment (platschi) [inline sdk/data/synths.ts:167]: We should keep Chainlink on L1 Ethereum (chainID 1) as it seems to be only deprecated on L2 (chainID 10) so far.

File: sdk/data/synths.ts [hunk 2]
```diff
@@ -40,15 +34,6 @@ type BasicSynth = {
 };
 
 export const synths: Record<SynthSymbol, BasicSynth> = {
-	sAAVE: {
-		name: 'Aave',
-		asset: 'AAVE',
-		addresses: {
-			1: '0xd2dF355C19471c8bd7D8A3aa27Ff4e26A21b4076',
```
Comment (platschi) [inline sdk/data/synths.ts:47]: We should keep Aave on L1 Ethereum (chainID 1) as it seems to be only deprecated on L2 (chainID 10) so far.

**Review by platschi — CHANGES_REQUESTED** [review]
See comments. The dropdown list, when scrolling to the very bottom, also keeps spinning/loading endlessly, it might be related to the github code scanning error (see warning under PR Files in github for https://github.com/Kwenta/kwenta/blob/257b34af1468ca1d7066cc706e62a51cdf9d08ac/sections/shared/modals/SelectCurrencyModal.tsx#L68

Screenshot 2023-02-03 at 17 06 56

File: sections/shared/modals/SelectCurrencyModal.tsx [hunk 3]
```diff
@@ -219,45 +221,6 @@ export const SelectCurrencyModal: FC<SelectCurrencyModalProps> = ({
...
-										symbol: synth.name,
-										isSynth: true,
-									}}
-								/>
-							);
-						})
-					) : (
-						<EmptyDisplay>{t('modals.select-currency.search.empty-results')}</EmptyDisplay>
-					)}
 					{oneInchTokensPaged.length ? (
```
Comment (LeifuChen) [inline sections/shared/modals/SelectCurrencyModal.tsx:224]: The infinite loading issue hasn't been fixed. Please check if the length of `oneInchTokensPaged` is different from `oneInchTokenList`.
image

**Review by LeifuChen — CHANGES_REQUESTED** [review]
Thanks for the work. Please note the following issue:
1. An infinite loader is at the bottom of the select currency modal.
2. All the synths are missing when switching to L1.
3. The category filter is not working.

File: sections/shared/modals/SelectCurrencyModal.tsx (hunk 3 above)
Comment (LeifuChen) [reply sections/shared/modals/SelectCurrencyModal.tsx:224]: This has been resolved after re-deployment.

**Review by platschi — APPROVED** [review]
Works now on L1 and L2 as requested :+1:

**Pull request merged**